thrift_rpc/logencoder.go

```diff
@@ -0,0 +1,60 @@
+package thrift_rpc
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/lightstep/lightstep-tracer-go/lightstep_thrift"
+	"github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+const (
+	deprecatedFieldKeyEvent   = "event"
+	deprecatedFieldKeyPayload = "payload"
```

Comment:
You can delete the original versions of these (though the "event" one is called "message")